lib/rspec/expectations/failure_aggregator.rb

```diff
@@ -150,11 +150,25 @@ def pluralize(noun, count)
       def enumerated(exceptions, index_offset)
         exceptions.each_with_index.map do |exception, index|
           index += index_offset
-          formatted_message = yield exception
+          formatted_message = "#{yield exception}\n#{format_backtrace(exception.backtrace).first}"
           "#{index_label index}#{indented formatted_message, index}"
         end
       end
 
+      def exclusion_patterns
+        %w[/lib\d*/ruby/ bin/ exe/rspec /lib/bundler/ /exe/bundle:].map! { |s| Regexp.new(s.gsub('/', File::SEPARATOR)) }
+      end
+
+      def format_backtrace(backtrace)
+        backtrace&.map { |l| backtrace_line(l) }&.compact&.tap { |filtered| filtered.concat backtrace if filtered.empty? }
```

Comment:
`&.` is not an operator we can use due to our supported Ruby versions, which is why this fails the build